active/website.md

```diff
@@ -0,0 +1,64 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Help to make the website accessible to all
```

Comment:
I wonder if this list and the delegated responsibilities could be a single list. Because I think the main responsibility is to make decisions on the direction of the website, and is a bit hidden in the second list under a bit more specific sounding "propose new features" which I think is selling it short a bit.